modules/installation-and-upgrade/nav-installation-and-upgrade-guide.adoc

```diff
@@ -117,10 +117,12 @@ ifeval::[{suma-content} == true]
 **** xref:proxy-y-z.adoc[Proxy - Minor Version or Patch Level Upgrade]
 endif::[]
 
+ifeval::[{suma-content} == true]
 *** xref:db-intro.adoc[Upgrade the Database]
 **** xref:db-migration-xy.adoc[Database Migration to Latest Version]
-*** xref:client-intro.adoc[Upgrade the Clients]
```

Comment:
Dont we also need to comment out the db-migration-xy.adoc file as well?

Reply:
in db-migration, we say:
```
...
* If you are running {opensuse} Leap 15.5, use PostgreSQL 15.
```
For the moment, I could change the PG version number to `14`.  Then it would fit, IMO.

Reply:
I created a separate patch to enable it again: https://github.com/uyuni-project/uyuni-docs/pull/2454

Reply:
But that PR is pointing to the branch for this PR... So @jcayouette, #2454 nees to be reviewed and merged first, and until that moment, the present PR **must not be merged**